libs/extensions/std/exec/src/gtfs-rt-interpreter-executor.ts

```diff
@@ -62,15 +101,17 @@ export class GtfsRTInterpreterExecutor
           resolve(this.parseTripUpdates(feedMessage, context));
           break;
         case 'alert':
-          // TODO: parse alert;
+          // Extract the alerts from thee feed message
+          resolve(this.parseAlerts(feedMessage, context));
           break;
         case 'vehicle':
-          // TODO: parse vehicle;
+          // Extract vehicle positions from thee feed message
+          resolve(this.parseVehiclePositions(feedMessage, context));
           break;
 
-        // No entitiy detected -> error
+        // Entity invalid
         default:
-          resolve(E.left(new Error('No entity detected in GTFS-RT-Feed. ')));
+          resolve(E.left(new Error('Entity invalid in GTFS-RT-Feed. ')));
```

Comment:
Which entity? Include more information like the entity type.

Reply:
changed to more meaningfull errormessage.